utils/gen_cursor_pngs.c

```diff
@@ -0,0 +1,42 @@
+#include <stdio.h>
```

Comment:
The file here not used and suppose it makes more sense if wanting to recreate the png files from the old headers. At the final stage the headers are removed so maybe could remove the file on the same commit as the headers.